code/backend/batch/utilities/helpers/EnvHelper.py

```diff
@@ -128,7 +128,7 @@ def __init__(self, **kwargs) -> None:
             "AZURE_FORM_RECOGNIZER_KEY"
         )
         # Azure App Insights
-        self.APPINSIGHTS_ENABLED = self.get_env_var_bool("APPINSIGHTS_ENABLED")
+        self.APPINSIGHTS_ENABLED = self.get_env_var_bool("APPINSIGHTS_ENABLED", "False")
```

Comment:
Is it worth adding a comment here to say that this will be turned on within an AppService when an App Insights is present?

Reply:
Added a comment, is that what you were looking for?